public/less/layout.less

```diff
@@ -31,21 +31,39 @@
     }
 }
 
-.nav-main-menu {
-    font-size: @font-size-large;
+header.navbar {
+    .logo{
+        padding-top: @padding-large-vertical;
+        padding-bottom: @padding-large-vertical;
+    }
 
-    ul.navbar-nav {
-        padding-top: @padding-xs-vertical;
-        border-bottom: none;
+    .nav-main-menu {
+        font-size: @font-size-large;
 
-        li a {
-            color: #fff;
-            font-weight: 300;
-        }
+        ul.navbar-nav {
+            padding-top: @padding-xs-vertical;
+            border-bottom: none;
 
-        li a:hover {
-            background-color: @main-menu-hover;
-        }
+            li a {
+                color: #fff;
+                font-weight: 300;
+            }
+
+            li a:hover {
+                background-color: @main-menu-hover;
+            }
 
+        }
     }
 }
+
+#application-container {
+    margin-bottom: 30px;
+}
+
+footer.navbar {
+    min-height: unset;
+    line-height: 1;
+    padding: 10px;
+    margin-bottom: 0;
```

Comment:
Added this because `.navbar` normally has a `margin-bottom` of `10px`, and we don't want a gap below the footer.